tests/sles4sap/publiccloud/qesap_ansible.pm

```diff
@@ -45,11 +45,10 @@ sub run {
         my $master_node = $self->get_promoted_hostname();
         $run_args->{site_a} = $instance if ($instance_id eq $master_node);
         $run_args->{site_b} = $instance if ($instance_id ne $master_node);
-        record_info("Instances:", "Detected HANA instances:
-        Site A: $run_args->{site_a}{instance_id}
-        Site B: $run_args->{site_b}{instance_id}") if ($instance_id eq $master_node);
     }
-
+    record_info("Instances:", "Detected HANA instances:
+    Site A (PRIMARY): $run_args->{site_a}{instance_id}
```

Comment:
Site A is not always PRIMARY

Reply:
It's initial state right after deployment,  before any test sequence is started.
Or is there a situation where you start with Primary being Site B?